tests/cmds/test_alerts.py

```diff
@@ -632,3 +632,27 @@ def test_search_with_or_query_flag_produces_expected_query(runner, cli_state):
     }
     actual_query = json.loads(str(cli_state.sdk.alerts.search.call_args[0][0]))
     assert actual_query == expected_query
+
+
+def test_send_to_makes_call_to_the_extract_method(
```

Comment:
some tests that validate that the `send_events` methods get called with the expected data would be helpful.

Reply:
added.